Core/scripts/dirac-install.py

```diff
@@ -1896,6 +1896,9 @@ def installExternals(releaseConfig):
   else:
     externalsVersion = releaseConfig.getExtenalsVersion()
```

Comment:
Is it really getExtenalsVersion and not getExternalsVersion ? The typo can be corrected then as well